### DISCUSSION TITLE
Fully reset remote DB

### DISCUSSION THREAD

**Discussion by thedevdavid** [supabase/supabase#23113] [Questions]
Is there a way to fully reset the remote DB? `supabase db reset --linked` only affects the `public` schema.

I'd love to be able to clear the unnecessary junk created in the database while testing with the team before going in production. Example: `auth` schema is full with audit logs, manually deleting test users is time consuming, etc.

**Comment by DylanGoodfellow** [comment]
This would be super handy since i have issues with the local dev instance of supabase - I use a remote instance for development and need to reset it semi frequently. it would be nice to have some consistency between the behaviour of  the command for a local project and one for a remote project on this.

**Comment by davegomez** [comment]
I came here with the same question, so it seems the only way is to destroy the project and recreate from scratch.

**Comment by Arputikos** [comment]
I would like to do the same too, since I had issues with auth migration done automatically that broke auth completely

**Comment by PidgeyBE** [comment]
I have the opposite, when I do `supabase db reset --linked`, my auth tables get totally wiped as well, while I want the opposite!
I want to keep my existing users...

**Comment by galex** [comment]
Same here, I am stuck with some trigger functions I cannot delete....

**Reply by j3h7sx** [reply]
Run this in the SQL Editor:

`DROP FUNCTION IF EXISTS public.[your-function]() CASCADE;`

(replace 'your-function')
